mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -37,6 +43,8 @@
 import okhttp3.Call;
 import okhttp3.Headers;
 
+import static android.os.Build.*;
```

Comment:
I think generic imports are against Google code style: https://source.android.com/setup/code-style#fully-qualify-imports

I'd only import android.os.Build non-statically.

Reply:
Ok, they are against. Changed it.